system-tests/services/test-pods.js

```diff
@@ -250,6 +250,261 @@ describe('Services', function () {
+        .root()
+        .get('input[name="env.3.key"]')
+        .type('UPPERCASE');
+
+      cy
+        .root()
+        .get('input[name="env.3.value"]')
+        .type('test');
+
+      cy
+        .get('label')
+        .contains('JSON Editor')
+        .click();
+
+      cy
+        .get('#brace-editor')
+        .contents()
+        .asJson()
+        .should('deep.equal', [{
+          'id': `/${Cypress.env('TEST_UUID')}/${serviceName}`,
+          'containers': [
+            {
+              'name': 'container-1',
+              'resources': {
+                'cpus': 0.1,
+                'mem': 10
+              },
+              'exec': {
+                'command': {
+                  'shell': command
+                }
+              }
+            }
+          ],
+          'scaling': {
+            'kind': 'fixed',
+            'instances': 1
+          },
+          'networks': [
+            {
+              'mode': 'host'
+            }
+          ],
+          'env': {
+            'camelCase': 'test',
+            'snake_case': 'test',
+            'lowercase': 'test',
+            'UPPERCASE': 'test'
+          }
+        }]);
+
+      cy
+        .get('button')
+        .contains('Review & Run')
+        .click();
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Service ID')
+        .contains(`/${Cypress.env('TEST_UUID')}/${serviceName}`);
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Instances')
+        .contains('1');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('CPU')
+        .contains('0.1 (0.1 container-1)');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Memory')
+        .contains('10 MiB (10 MiB container-1)');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Disk')
+        .contains('Not Supported');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('GPU')
+        .contains('Not Supported');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Container Image')
+        .contains('Not Configured');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Force Pull On Launch')
+        .contains('Not Configured');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('CPUs')
+        .contains('0.1');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Memory')
+        .contains('10 MiB');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Command')
+        .contains(command);
+
+      cy.viewport(1000, 2000);
```

Comment:
Yeah, it seems that the `testing/master` cluster has the environment variables missing. It seems to be there on the current master 👍